aireview/src/aireview/infrastructure/github_client.py

```diff
@@ -56,27 +58,33 @@ async def submit_review(
     ) -> None:
         response: Response
         try:
+            logger.info(f"Submit review to pull request #{pr_number}")
             for comment in review.comments:
-                review_data = {
-                    "body": f"**{comment.type}**: {comment.content}",
-                    "commit_id": '',
-                    "path": comment.file_path,
-                    "start_line": comment.line,
-                    "start_side": 'RIGHT',
-                    "line": comment.line+1,
-                    "side": 'RIGHT'
-                }
-
-                response = self._session.post(f"{self._base_url}/repos/{self.owner}/{self.repo}/pulls/{pr_number}/comments",headers=self._headers, json=review_data)
-                response.raise_for_status()
+                logger.info(f"comment: {comment}")
+                content = self.get_file_content(comment.file_path, str(pr_number))
+                if content is not None:
+                    if self.len_content_lines(content) <= int(comment.line):
+                        comment.line = self.len_content_lines(content) - 1
+                    review_data = {
+                        "body": f"**{comment.type}**: {comment.content}",
+                        "commit_id": f"951b12a93e1ade83bce9d18c4d240bfa2f9d3067",
+                        "path": comment.file_path,
```

Comment:
**style**: Hardcoded commit ID '951b12a93e1ade83bce9d18c4d240bfa2f9d3067' should be passed as a parameter or retrieved dynamically